tests/spec/w3c/headers-spec.js

```diff
@@ -221,6 +221,25 @@ describe("W3C — Headers", function() {
     });
   });
 
+  describe("title", () => {
+    it("uses <h1> if already present", async () => {
+      const ops = makeStandardOps();
```

Comment:
I think we are missing a test checking the effect on `conf.title`. 

```JS
const { title } = doc.defaultView.respecConfig; 
expect(title).toEqual(...);
```

Reply:
note that here we want to be sure we hit both code paths for the if statement. So, we might actually need two independent tests: 

1. one with the `respecConfig.title` and `h1`
1. one without the `respecConfig.title` but with `h1`

Same applies to `subtitle`.

Reply:
Even I was thinkning about the same. But in the case of `subtitle` we do have a test for a missing `subtitle` and moreover the subtitle element is not mandatorily present on the doc. So, do we still need it in the case of `subtitile` ?

Reply:
> ...not mandatorily present on the doc. So, do we still need it in the case of subtitile ?

So, mostly we are interested in making sure we walk all code paths, and think of weird edge cases. Although we'd like to assume things about what will be common usage, things break when users do things we don't expect.

Reply:
To be clear, I'm particularly interested in this being tested:
https://github.com/w3c/respec/pull/1516/files#diff-ebb1a497c097951bb868645bd73ade4fR29

Reply:
> we want to be sure we hit both code paths for the if statement

@marcoscaceres you mean one with `h1` and one without `h1`? and not (w/ and w/o `respecConfig.title`)
so that our tests will be "*uses `respecConfig.title` if `<h1>` not present*" and "*uses `<h1>` for title if already present*"?

Reply:
> To be clear, I'm particularly interested in this being tested:
https://github.com/w3c/respec/pull/1516/files#diff-ebb1a497c097951bb868645bd73ade4fR29

I guess we can test for this here : 

https://github.com/shikhar-scs/respec/blob/e3ddc6f13af4d4a965f4b5f22944dda35e895caf/tests/spec/w3c/headers-spec.js#L270-L281

Reply:
It's best to just leave the old tests and just add new ones (even if they overlap). Changing existing tests risks introducing new regressions.

Reply:
Ok I'll implement that. Also @marcoscaceres is there any way I could run only a single test. I mean I have to go to `http://localhost:8080/tests/SpecRunner.html` after every little change I make (and then it starts all over again ). So, is there any way I could run just a single test or test file ?

Reply:
Ah yeah, just put "f" in front of either `it(` or `describe(`. So:

```
fit();// "focus" this one
fdescribe(); // focus all these tests. 
```

You an also click spec in the results page, to only run the files associated with that spec.

Reply:
More info https://jasmine.github.io/2.1/focused_specs.html

Reply:
This is a life saviour. Earlier I used to sit for minutes just to see small changes. Thanks Marcos

Reply:
Feel your pain. Just remember to remove the `fdescribe` and `fit` when done. I've accidentally checked in a few of those ☺️.

Reply:
Ok so this has been covered :+1: